app/apptesting/test_suite.go

```diff
@@ -350,7 +348,6 @@ func (s *KeeperTestHelper) SetupValidator(bondStatus stakingtypes.BondStatus) sd
 	signingInfo := slashingtypes.NewValidatorSigningInfo(
 		consAddr,
 		s.Ctx.BlockHeight(),
-		0,
 		time.Unix(0, 0),
 		false,
```

Comment:
I'm worried this means that my state breaking commit got into the version were importing -- fine for main, not for v23.x

Reply:
No thats what my comment is above

> NOTE: When backporting, some of these changes should be reverted, because this bumps the cosmos sdk fork version for main, which includes state breaking changes. So we need to ensure when backporting we use the respective v23 sdk branches.